### ISSUE TITLE
Ember.js Files cannot be created.

### ISSUE THREAD

**Issue by jmochenkova** [Turbo87/intellij-emberjs#216]
WebStorm 2018.2 EAP (182.3569.4) + Ember.js plugin ver. 2018.2.2-0
Create Ember project. Invoke "New - Ember.js Files". Select any kind, set name and press OK.
The exception will be thrown, no file will be created.

```
java.lang.NoClassDefFoundError: org/apache/commons/lang3/SystemUtils
	at com.emberjs.cli.EmberCli.commandLine(EmberCli.kt:14)
	at com.emberjs.cli.EmberCli.run(EmberCli.kt:28)
	at com.emberjs.cli.EmberCliGenerateTask.run(EmberCliGenerateTask.kt:40)
	at com.intellij.openapi.progress.impl.CoreProgressManager$TaskRunnable.run(CoreProgressManager.java:736)
	at com.intellij.openapi.progress.impl.CoreProgressManager$5.run(CoreProgressManager.java:434)
	at com.intellij.openapi.progress.impl.CoreProgressManager.a(CoreProgressManager.java:157)
	at com.intellij.openapi.progress.impl.CoreProgressManager.a(CoreProgressManager.java:580)
	at com.intellij.openapi.progress.impl.CoreProgressManager.executeProcessUnderProgress(CoreProgressManager.java:525)
	at com.intellij.openapi.progress.impl.ProgressManagerImpl.executeProcessUnderProgress(ProgressManagerImpl.java:85)
	at com.intellij.openapi.progress.impl.CoreProgressManager.runProcess(CoreProgressManager.java:144)
	at com.intellij.openapi.application.impl.ApplicationImpl.a(ApplicationImpl.java:570)
	at com.intellij.openapi.application.impl.ApplicationImpl$1.run(ApplicationImpl.java:310)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
```

**Comment by Turbo87** [comment]
@makepanic this seems related to the latest changes

**Comment by makepanic** [comment]
I can reproduce it if ember-cli has an issue before running the selected task.

Example:
* trying to generate a file before having node_modules/bower_components installed

I'm not sure if there's anything we can do except improve error reporting in this case. Looks like the error's getting swallowed

**Comment by Turbo87** [comment]
`java.lang.NoClassDefFoundError: org/apache/commons/lang3/SystemUtils` sounds more like a wrong/broken import path to me 🤔

**Comment by Turbo87** [comment]
@jmochenkova should hopefully be fixed by v2018.2.3 which will be published in the next couple of minutes. thanks for reporting! :)

**Comment by jmochenkova** [comment]
@Turbo87 Thanks) It works with the plugin ver. 2018.2.3-0
